File: app/src/main/java/com/example/capture/MusicApplication.java
package com.example.capture;

import android.app.Application;

public class MusicApplication extends Application {
    private static MusicApplication mInstance;
    private MusicServiceInterface mInterface;

    @Override
    public void onCreate() {
        super.onCreate();
        mInterface = new MusicServiceInterface(getApplicationContext());
        mInstance = this;
    }

    public static MusicApplication getInstance() {
        return mInstance;
    }

    public MusicServiceInterface getServiceInterface() {
        return mInterface;
    }
}

File: app/src/main/java/com/example/mediaplayer/MusicApplication.kt
package com.example.mediaplayer

import android.app.Application

class MusicApplication : Application() {
    private var mInterface: MusicServiceInterface? = null
    override fun onCreate() {
        super.onCreate()
        mInterface = MusicServiceInterface(applicationContext)
        mInstance = this
    }

    fun getServiceInterface(): MusicServiceInterface? {
        return mInterface
    }

    companion object {
        private var mInstance: MusicApplication? = null
        fun getInstance(): MusicApplication? {
            return mInstance
        }
    }
}